README.md

```diff
@@ -222,6 +222,16 @@ export default {
 };
 ```
 
+##### With [`Netlify functions`](https://docs.netlify.com/functions/overview/)
+
+```js
+import { createHandler } from 'graphql-http/lib/use/netlify'; // npm install graphql-http
+import { schema } from './previous-step';
+
+// Create the GraphQL over HTTP native fetch handler
+export const handler = createHandler({ schema });
```

Comment:
So you use it by just exporting the handler?

Reply:
yes, the signature is quite simple in this way. I made it so that `createHandler` returns a full `Handler` fn instance as described by the netlify types. I'm going to do a global yarn link and test it manually with `netlify-cli` just to be sure